src/aviary/utils.py

```diff
@@ -149,3 +173,182 @@ async def eval_answer(
         return float(gt in pred)
 
     raise RuntimeError(f"Invalid evaluation mode: {eval_mode}")
+
+
+_CAPITAL_A_INDEX = ord("A")
+
+
+class MultipleChoiceQuestion(BaseModel):
+    QUESTION_PROMPT_TEMPLATE: ClassVar[str] = "Q: {question}\n\nOptions:\n{options}"
+    # TODO: combine with above eval_answer and its prompts
+    EVALUATION_PROMPT_TEMPLATE: ClassVar[str] = (
+        "Given the following question and a proposed answer to the question, return the"
+        " single-letter choice in the question that matches the proposed answer."
+        " If the proposed answer is blank or an empty string,"
+        " or multiple options are matched, respond with '0'."
```

Comment:
Maybe this should be something a little more distinct than "0" only because I've seen some confusing circumstances where the correct letter answer is "0" i.e. ( `A.) 0`). And the system might get confused via the prompting in that circumstance. Could be like `"NO_MATCH"` or something.

Reply:
So `0` is outside of `A-Z` so there's no way to confuse, where as `NO_MATCH` could be interpreted as `N`.

In general, ima move to Andrew's prompting strategy from https://github.com/Future-House/aviary/pull/148, so ima resolve this in the forthcoming PR

Reply:
https://github.com/Future-House/aviary/pull/161 removes this `0` thing, so resolved there